src/components/ADempiere/Form/VPOS/Collection/overdrawnInvoice/index.vue

```diff
@@ -0,0 +1,278 @@
+        <el-button
+          type="danger"
+          class="custom-button-create-bp"
+          icon="el-icon-close"
+          @click="close"
+        />
+        <el-button
+          type="primary"
+          class="custom-button-create-bp"
+          icon="el-icon-check"
+          @click="success"
+        />
+      </span>
+    </el-dialog>
+  </div>
+</template>
+
+<script>
+import { formatPrice } from '@/utils/ADempiere/valueFormat.js'
+import formMixin from '@/components/ADempiere/Form/formMixin'
+import posMixin from '@/components/ADempiere/Form/VPOS/posMixin.js'
+import fieldsListOverdrawnInvoice from './fieldsListOverdrawnInvoice.js'
+import { overdrawnInvoice } from '@/api/ADempiere/form/point-of-sales.js'
+
+export default {
+  name: 'OverdrawnInvoice',
+  mixins: [
+    formMixin,
+    posMixin
+  ],
+  props: {
+    change: {
+      type: Number,
+      default: 0
+    },
+    pay: {
+      type: Number,
+      default: 0
+    },
+    pending: {
+      type: Number,
+      default: 0
+    },
+    totalOrder: {
+      type: Number,
+      default: 0
+    },
+    currency: {
+      type: Object,
+      default: undefined
+    },
+    metadata: {
+      type: Object,
+      default: () => {
+        return {
+          uuid: 'OverdrawnInvoice',
+          containerUuid: 'OverdrawnInvoice'
+        }
+      }
+    }
+  },
+  data() {
+    return {
+      option: 1,
+      fieldsList: fieldsListOverdrawnInvoice
+    }
+  },
+  computed: {
+    showDialogo() {
+      return this.$store.state['pointOfSales/payments/index'].dialogoInvoce.show
+    },
+    caseOrder() {
+      return this.$store.state['pointOfSales/payments/index'].dialogoInvoce.type
+    }
+  },
+  methods: {
+    formatPrice,
+    success() {
+      const customerDetails = []
+      this.fieldsList.forEach(element => {
+        const value = this.$store.getters.getValueOfField({
+          containerUuid: 'OverdrawnInvoice',
+          columnName: element.columnName
+        })
+        customerDetails.push({
+          label: element.columnName,
+          value
+        })
+      })
+      const posUuid = this.currentPointOfSales.uuid
+      const orderUuid = this.$route.query.action
+      overdrawnInvoice({
+        posUuid,
+        orderUuid,
+        createPayments: !this.isEmptyValue(this.currentOrder.listPayments.payments),
+        payments: this.currentOrder.listPayments.payments,
+        customerDetails,
+        option: this.option
+      })
+        .then(response => {
+          this.$store.dispatch('reloadOrder', response.uuid)
+          this.$message({
+            type: 'success',
+            message: this.$t('notifications.completed'),
+            showClose: true
+          })
+        })
+        .catch(error => {
+          this.$message({
+            type: 'error',
+            message: error.message,
+            showClose: true
+          })
+        })
+      this.$store.commit('dialogoInvoce', { show: false, success: true })
```

Comment:
please change mutation name to `dialogInvoice` (without the `o` at the end of the dialog)